Makefile

```diff
@@ -1,8 +1,12 @@
 SHELL := /bin/bash
 
-run:
+run: sendgrid-init
 	cd $(PROJECT_DIR) && AUTO_APPROVE="-auto-approve" make
 
+sendgrid-init: 
```

Comment:
Should this be in the templated makefile instead?

Reply:
That's where i wanted to put it as well 
but I need the `zero apply`'s env with the `sendgridApiKey`

Reply:
Maybe we could push that into an aws secret during the secret init step, then have the script pull it out?

Reply:
Yeah lets do that instead 👍